### PR TITLE
[MM-28140] e2e/system_console/customization: add T1027

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
Adds T1027: Custom branding is enabled but no image has been uploaded

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28140

### CODE REVIEW

File: e2e/cypress/support/api/system.js [hunk 1]
```diff
@@ -152,3 +152,16 @@ function uploadLicenseIfNotExist() {
         });
     });
 }
+
+Cypress.Commands.add('apiDeleteBrandImage', () => {
```
Comment (isacikgoz) [inline e2e/cypress/support/api/system.js:156]: I wonder these should be included into UI tests. I mean, it seems like we are skipping the system console test here.

Comment (mgdelacroix) [reply e2e/cypress/support/api/system.js:156]: I added it like a function as it was more a precondition of the test than an action (make sure no image has been uploaded yet VS remove the uploaded image)

Comment (saturninoabril) [reply e2e/cypress/support/api/system.js:156]: Thanks for bringing up @isacikgoz but I agree with @mgdelacroix. We'll cover such case in [MM-T1030](https://automation-test-cases.vercel.app/test/MM-T1030).

Comment (isacikgoz) [reply e2e/cypress/support/api/system.js:156]: Sounds good.

File: e2e/cypress/integration/system_console/ui_and_api/customization_spec.js [hunk 2]
```diff
@@ -396,6 +396,27 @@ describe('Customization', () => {
             expect(config.TeamSettings.EnableCustomBrand).to.equal(false);
         });
     });
+
+    it('T1027 - Custom branding is enabled but no image has been uploaded', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/system_console/ui_and_api/customization_spec.js:400]:
```suggestion
    it('MM-T1027 - Custom branding is enabled but no image has been uploaded', () => {
```

**Review by isacikgoz — APPROVED** [review]
Nice one! 🎉

File: e2e/cypress/support/api/system.js [hunk 3]
```diff
@@ -152,3 +152,16 @@ function uploadLicenseIfNotExist() {
...
+    return cy.request({
+        headers: {'X-Requested-With': 'XMLHttpRequest'},
+        url: '/api/v4/brand/image',
+        method: 'DELETE',
+        failOnStatusCode: false,
+    }).then((response) => {
+        // both deleted and not existing responses are valid
+        expect(response.status).to.be.oneOf([200, 404]);
+        return cy.wrap({response});
```
Comment (saturninoabril) [inline e2e/cypress/support/api/system.js:165]: For consistency if not returning specific object.
```suggestion
        return cy.wrap(response);
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @mgdelacroix! LGTM, except for couple requests and move to other folder.

**Comment by saturninoabril** [comment]
@mgdelacroix Ping me when ready for re-review.

**Review by saturninoabril — APPROVED** [review]
Thanks @mgdelacroix, LGTM.

**Review by lindy65 — APPROVED** [review]
Thanks @mgdelacroix - looks good to me 👍

**Pull request merged**